### PR TITLE
add NOTICE file

### PR DESCRIPTION
**Pull request by christo4ferris**
#### Type of change

- Documentation update

#### Description

A NOTICE file should be included per ASF policy. Content per ASF
policy adapted to Hyperledger.

Signed-off-by: Christopher Ferris chrisfer@us.ibm.com

#### Additional details

doc only change

#### Related issues

none

### CODE REVIEW

**Review by christo4ferris — COMMENTED** [review]
You don't put a license header in the NOTICE file... pls fix the linter

**Comment by lindluni** [comment]
https://github.com/hyperledger/fabric/blob/master/scripts/functions.sh

Add an entry here to ignore it

File: NOTICE [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Hyperledger Fabric
+Copyright [2016-2020] contributors to Hyperledger Fabric
+
+This product includes software developed at
```
Comment (mastersingh24) [inline NOTICE:4]: This actually does not make sense to me.  What code was actually developed at or by Hyperledger?

**Comment by lindluni** [comment]
Mistakenly closed, but reopened

**Comment by CsterKuroi** [comment]
Copyright？

File: NOTICE (hunk 1 above)
Comment (mastersingh24) [reply NOTICE:4]: I see that ASF does it this way as well.   Still seems odd to me.

**Comment by lindluni** [comment]
@christo4ferris Can you please ammend this PR to include an exclusion for the NOTICE file here: https://github.com/hyperledger/fabric/blob/master/scripts/functions.sh

**Comment by lindluni** [comment]
@christo4ferris I pushed the fix for the license check into your branch, but your commit is missing DCO. Can you pull the change I made and amend your commit with your signature.

**Comment by ale-linux** [comment]
@christo4ferris please add the DCO so we can merge it, this PR has been out for a while

**Comment by lindluni** [comment]
I addressed this in #1333 so we can close this

**Pull request closed without merging**